spec/book_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../classes/book'
+
+describe Book do
+  context 'Testing the Book Class'
+
+  title = 'The Lord of the Rings'
+  author = 'J.R.R. Tolkien'
+  publish_date = '1954-07-29'
+  publisher = 'Allen & Unwin'
+  cover_state = 'good'
+  book = Book.new(title, author, publisher, cover_state, publish_date)
+
```

Comment:
- [ ] Kindly add a spec for `can_be_archived?()` in the _Book_ class with the following conditions:
    * should override the method from the parent class
    * should return true if the parent's method returns true OR if cover_state equals "bad"
    * otherwise, it should return false